ui/admin/tests/acceptance/accounts/delete-test.js

```diff
@@ -63,14 +63,14 @@ module('Acceptance | accounts | delete', function (hooks) {
   });
 
   test('can delete an account', async function (assert) {
-    const accountsCount = this.server.schema.accounts.all().models.length;
+    const accountsCount = this.server.db.accounts.length;
```

Comment:
Tests failed on `cannot read properties of 'schema'` so I tested a fix to replace with `server.db` and this works. Please note that there is a ticket to cleanup our usage of `server.db` - and there was also a discussion about using schema vs db [here](https://hashicorp.slack.com/archives/C07EMPECGFN/p1739372121647269?thread_ts=1739371374.513759&cid=C07EMPECGFN). TLDR "...we should be using schema unless you need the db for a specific reason"